core/src/main/scala/kafka/tools/StreamsResetter.java

```diff
@@ -316,6 +317,7 @@ private void maybeResetInputAndSeekToEndIntermediateTopicOffsets(final Map consu
                 for (final String topic : notFoundInputTopics) {
                     System.out.println("Topic: " + topic);
                 }
+                topicNotFound = 1;
```

Comment:
topicNotFound sounds like a boolean.
Its value is either 0 or 1.
Declaring topicNotFound as boolean is better.